index.html

```diff
@@ -95,11 +95,24 @@
               </select>
             </td>
             <td ng-if="sectionName==='taken'">
-              <select ng-options="type as capitalize(type) for type in DAMAGE_TYPES"
-                      ng-model="value.types"
-                      ng-change="updateTotal(table)"
-                      class="input" multiple>
-              </select>
+              <table class="multiSelect">
+                <tr>
+                  <td><label class="select physical" ng-class="{selected: value.types.physical}">
+                    <input type="checkbox" ng-model="value.types.physical" ng-change="add(table)"></label></td>
```

Comment:
This was a test to see if it correctly updated the total on change, which didn't work. Right now you have to update the number value to test the changes.